### PR TITLE
Add treanz-Chernivtsi file

### PR DESCRIPTION
**Pull request by treanz**
this is a beginning of the Chernivtsi file 🌞  🐈 
 
:wall_of_text:

### CODE REVIEW

**Comment by shinton** [comment]
Hey, you're very far away! Here [wikipedia](https://en.wikipedia.org/wiki/Chernivtsi), right?

**Comment by treanz** [comment]
You are absolutely right 😄 . It is beautiful city, -  welcome !

File: treanz-Chernivtsi.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Chernivtsi, Cv
+
+### Restaurants
+ - New York Pizza
+ - etc
+
+### Local Attractions
+ - Chernivtsi National University
+ - some Church
```
Comment (treanz) [inline treanz-Chernivtsi.md]: i don't remember label

**Comment by fluyapstone** [comment]
:shipit:

**Pull request merged**